fossunited/api/schedule.py

```
@@ -97,7 +107,338 @@ def get_event_schedule(event_id: str) -> dict:
+            continue
+        if halls_list and s.hall not in halls_list:
+            continue
+
+        s.start_time_str = str(timedelta(seconds=int(s.start_time.total_seconds())))
+        s.end_time_str = str(timedelta(seconds=int(s.end_time.total_seconds())))
+        s.speakers_list = []
+
+        if s.speakers:
+            try:
+                s.speakers_list = json.loads(s.speakers)["speakers"]
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)